mita/openstack.py

```diff
@@ -220,7 +220,7 @@ def _get_network(self, network: Optional[str] = None) -> OpenStackNetwork:
                 logger.debug(be)
                 continue
 
-        raise ResourceNotFound(f"No suitable network resource found.")
+        raise ResourceNotFound("No suitable network resource found.")
```

Comment:
One commit edits this `ResourceNotFound` string to be an f-string, and then another commit in this same PR removes the `f`. Can you rebase your branch and edit the original commit so that no commits change this line? Then it will be easier to review each commit individually.

Reply:
I actually swept it under the rug... initially, the argument to the method was the network id hence didn't have the resource name. Subsequent changes forced me to get the Network object... sorry, I should have made it easier.